.github/workflows/build.yml

```
@@ -6,6 +6,8 @@ on:
     branches:
       # Push events on the main branch
       - main
+      - release/**
+      - eculver/**
```

maybe we can change this to something more generic ?
```suggestion
      - test-release/**
```

---

Oh, that was just to show the crowd (and for my own satisfaction) that the pattern matching was doing what it was supposed to. I removed mine from the list.